sequence_task_1/student/include/impl.h

```diff
@@ -8,33 +8,71 @@ class StaticPlaylist
 {
 public:
     /** @todo Member traits */
+    typedef Song_t value_type;
```

Comment:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#t43-prefer-using-over-typedef-for-defining-aliases